doc/refman/RefMan-tac.tex

```diff
@@ -2688,6 +2703,68 @@ \subsection{\tt inversion \ident}
 Note that \texttt{H} has been substituted by \texttt{(LeS n m0 l)} and
 \texttt{m} by \texttt{(S m0)}.
 
+\example{Using \texorpdfstring{\texttt{inversion\_sigma}}{inversion\_sigma}}
+
+Let us consider the following inductive type of length-indexed lists,
+and a lemma about inverting equality of \texttt{cons}:
+
+\begin{coq_eval}
+Reset Initial.
+Require Coq.Logic.Eqdep_dec.
+\end{coq_eval}
+
+\begin{coq_example*}
+Inductive vec A : nat -> Type :=
+| nil : vec A O
+| cons {n} (x : A) (xs : vec A n) : vec A (S n).
+
+Lemma invert_cons : forall A n x xs y ys,
+    @cons A n x xs = @cons A n y ys
+    -> xs = ys.
+Proof.
+intros A n x xs y ys H.
+\end{coq_example*}
+
+\begin{coq_example}
+Show.
```

Comment:
No reason to insert `Show.` here. Just put move the previous tactic from the `coq_example*` block to this `coq_example` one.